sunpy/coordinates/ephemeris.py

```diff
@@ -74,37 +126,72 @@ def get_body_heliographic_stonyhurst(body, time='now', observer=None):
             ltt_string = f"{light_travel_time.to_value('s')}"
         log.info(f"Apparent body location accounts for {ltt_string} seconds of light travel time")
 
+    if include_velocity:
+        pos, vel = get_body_barycentric_posvel(body, emitted_time)
+        body_icrs = pos.with_differentials(vel.represent_as(CartesianDifferential))
+    else:
+        body_icrs = get_body_barycentric(body, emitted_time)
+
     body_hgs = ICRS(body_icrs).transform_to(HeliographicStonyhurst(obstime=obstime))
 
     return body_hgs
 
 
 @add_common_docstring(**_variables_for_parse_time_docstring())
-def get_earth(time='now'):
+def get_earth(time='now', include_velocity=False):
     """
     Return a `~astropy.coordinates.SkyCoord` for the location of the Earth at a specified time in
-    the `~sunpy.coordinates.frames.HeliographicStonyhurst` frame.  The longitude will be 0 by definition.
+    the `~sunpy.coordinates.frames.HeliographicStonyhurst` frame.  The longitude will be zero by
+    definition.
 
     Parameters
     ----------
     time : {parse_time_types}
         Time to use in a parse_time-compatible format
+    include_velocity : `bool`
+        If True, include the Earth's velocity in the output coordinate.  Defaults to False.
 
     Returns
     -------
     out : `~astropy.coordinates.SkyCoord`
         Location of the Earth in the `~sunpy.coordinates.frames.HeliographicStonyhurst` frame
+
+    Notes
+    -----
+    The Earth's velocity in the output coordinate will invariably be negligible because the
```

Comment:
```suggestion
    The Earth's velocity in the output coordinate will invariably be zero because the
```
?

Reply:
The velocity should be zero, but it generally won't be *exactly* zero because:

- The rotation of the frame is calculated via finite-difference methods on the positions of Earth, which will always have some inaccuracy compared to the actual velocity of Earth.
- For the built-in Astropy ephemeris, the returned velocity is inaccurate compared to the derivative of the returned positions.  I think this is documented somewhere.  The JPL ephemeris is fine.
- Unavoidable numerical noise

That's why I'm reluctant from sounding like it's a promise that it will be zero.  I also don't want to force it to be zero because that may lead to inconsistencies.